### PR TITLE
Support `@use` for builtin modules

### PR DESCRIPTION
**Pull request by kaj**
A part of #60 .

This PR only deals with builtin modules.

- [x] Parse the `@use` statement to an `Item`.
- [x] Provide all the builtin modules.
- [x] Use correct names for existing functions in the modules and in the global scope (some differ).
- [ ] Provide all expected functions in the standard module, or note exceptions in #60.

### CODE REVIEW

File: src/functions/math.rs [hunk 1]
```diff
@@ -242,6 +268,12 @@ fn number(v: Rational, unit: Unit) -> Value {
     Value::Numeric(Number::from(v), unit, true)
 }
 
+/// convert f64 in radians (used by rust) to numeric Value in degrees
+/// (used by sass).
+fn deg_value(rad: f64) -> Value {
+    float_value(rad * 180. / std::f64::consts::PI, Unit::Deg)
```
Comment (glebm) [inline src/functions/math.rs:274]: There is https://doc.rust-lang.org/std/primitive.f64.html#method.to_degrees

Comment (kaj) [reply src/functions/math.rs:274]: Ah, thanks!

**Comment by kaj** [comment]
I'm considering merging this without further improvements to the actual functions.  That can always be done later.

Any thoughts on the "module system" itself?  I know that it is rather sketchy and only handles functions, so I think a `Module` should be a struct and not just a type alias for a map.  But that may also be an area for future improvements.

**Comment by glebm** [comment]
This is the first Sass implementation to begin implementing modules, so this is uncharted territory!

Doesn't have to be perfect before merging :+1:

**Pull request merged**